pkgs/top-level/python-packages.nix

```diff
@@ -225,6 +225,8 @@ in {
 
   diff-match-patch = callPackage ../development/python-modules/diff-match-patch { };
 
+  flootty = callPackage ../development/python-modules/flootty { };
```

Comment:
This does not look like a python library to me. If that is the case please move it out of this module as we build `pythonPackages` against multiple python versions.

Reply:
Should I also move the derivation? If so, what do you think is the right place? development/tools? applications/networking? Somewhere else?

Reply:
`development/tools` sounds good.